apps/settings/js/modules/apn/apn_list.js

```diff
@@ -177,7 +177,7 @@ define(function(require) {
      *
      * @access private
      * @memberOf ApnList.prototype
-     * @returns {Promise Array<ApnItem>}
+     * @returns {(Promise|Array<ApnItem>)}
```

Comment:
Just realized that we may have to change `Array<ApnItem>` to `Array.<ApnItem>`. Following the previous comment the entire thing should be `Promise.<Array.<ApnItem>>`.